{{cookiecutter.project_name}}/.github/workflows/changelog.yml

```diff
@@ -13,6 +13,6 @@ on:
 
 jobs:
   call-changelog-check-workflow:
-    uses: ASFHyP3/actions/.github/workflows/reusable-changelog-check.yml@main
+    uses: ASFHyP3/actions/.github/workflows/reusable-changelog-check.yml@v0.7.0
     secrets:
-      USER_TOKEN: {{'${{ secrets.GITHUB_TOKEN }}'}}
```

Comment:
So the `{{'...'}}` shenanigans is because cookiecutter uses jinja2 as the templating engine, and GitHub actions *also* double `{{}}` for their templating. You'll need to "escape" all the GitHub actions brackets for jina2

Reply:
They should all be fixed.